client/src/components/home/Home.js

```diff
@@ -9,36 +9,61 @@ import Veggies from "./img/veggies_thinking.png";
 
 function Home(props) {
     return (
-        <Grid columns={"2"} verticalAlign={"middle"} textAlign={"center"} stackable>
+        <Grid columns={"2"} verticalAlign={"middle"} stackable>
+            {/*Hero banner*/}
             <Grid.Row className="hero">
-                <Grid.Column>
-                    <Header>Do I live in a food desert?</Header>
+
+                <Grid.Column className="info-left info">
+                    <Header>Do I live in an Atlanta food desert?</Header>
+                    <p className="hero-text">Scroll down to learn more about how <i className="emph"> where you live in Atlanta can affect the food you eat.</i></p>
+                    <Button className="main-cta cta">View our map</Button>
```

Comment:
You can make this take the user to the wizard by adding an `as={Link}` (`Link` is a Semantic UI React component, so make sure you import it) and a `to` prop in React with a value of `/map`

Also better button text might be along the lines of "find out now"

Reply:
Got it!